taskcat/cfg/schema_legacy_project_config.json

```diff
@@ -20,8 +20,11 @@
                                 "pattern": "^(ap|eu|us|sa|ca|cn|af|me|us-gov)-(central|south|north|east|west|southeast|southwest|northeast|northwest)-[0-9]$"
                             }
                         },
-                        "parameter_input_file": {"type": "string" },
-                        "parameters": { "$ref": "schema_definitions.json#/definitions/parameters" }
+                        "parameter_input_file": { "oneOf": [ {"type": "null"}, {"type": "string"} ] },
+                        "parameters": { "$ref": "schema_definitions.json#/definitions/parameters" },
+                        "auth": {"type": "object"},
```

Comment:
We don't support auth in legacy project configs. What's the intent here?

Reply:
no specific intent, was simpler and didn't see a reason not to allow it in legacy configs